app/models/miq_server.rb

```diff
@@ -608,4 +608,8 @@ def server_timezone
   def tenant_identity
     User.super_admin
   end
+
+  def objects_in_settings_chain
+    zone ? zone.objects_in_settings_chain << self : [self]
```

Comment:
Does calling this method mutate zone/region's data by pushing itself onto it? Is it idempotent?